modules/tensor_mechanics/src/materials/IsotropicPlasticityStressUpdate.C

```diff
@@ -24,8 +24,8 @@ InputParameters validParams<RecomputeRadialReturnIsotropicPlasticity>()
 }
 
 
-RecomputeRadialReturnIsotropicPlasticity::RecomputeRadialReturnIsotropicPlasticity(const InputParameters & parameters) :
-    RecomputeRadialReturn(parameters),
+IsotropicPlasticityStressUpdate::IsotropicPlasticityStressUpdate(const InputParameters & parameters) :
+    RadialReturnStressUpdate(parameters),
     _yield_stress_function(isParamValid("yield_stress_function") ? &getFunction("yield_stress_function") : NULL),
     _yield_stress(getParam<Real>("yield_stress")),
```

Comment:
What? No. Right now the user can only specify a *_constant or a *_function. _Both_ for hardening _and_ yield. What I'm saying is that a constant is a simple form of a function! Just have one parameter and make it a function. The user can still put `yield_stress = 3` or `yield_stress = my_awesome_function` in their input files!